pkg/server/server.go

```diff
@@ -303,6 +304,17 @@ func (h *handler) authenticateEndpoint(w http.ResponseWriter, req *http.Request)
 		"method": req.Method,
 	})
 
+	if logrus.IsLevelEnabled(logrus.DebugLevel) {
+		reqDump, err := httputil.DumpRequest(req, true)
+		if err != nil {
+			log.Info(err, "Failed to dump http request")
+		}
+
+		log.WithFields(logrus.Fields{
+			"request": string(reqDump),
```

Comment:
are we sure we want this? It'll dump all tokens to logs

Reply:
Good point... I'll update this to only log safe fields from the request